web/README.md

```diff
@@ -106,6 +115,6 @@ Run:
 ```
 dotnet test tests\Web.A11yTests
 ```
-
+[comment]: <> (How does Playwright come into this? I haven't seen a reference yet in this document?)
```

Comment:
It's to raise awareness that Playwright is being used - there isn't anything that is needed, as its installed as a nuget dependency

Reply:
Should we add a brief description (a line or so) on what it is being used for for those not familiar with it?

Reply:
Feels like a can of worms - there's many libraries in the solution that people might not be familiar, there has to be an assumed amount that people will research knowledge gaps. We could add links in on the additional resources page if we feel it necessary?

The line about Playwright was only called out to raise awareness. The testing technically choices in the is cover in the testing documentation.